test/WebJobs.Script.Tests/Management/FunctionsSyncManagerTests.cs

```diff
@@ -438,6 +442,31 @@ public void Use_Website_Name_If_Website_Hostname_Is_Not_Available(string envKey,
             }
         }
 
+        [Fact]
+        public async Task Logging_ConfiguredCorrectly()
+        {
+            _scriptSettingsManagerMock.Setup(p => p.GetSetting(EnvironmentSettingNames.AzureWebsiteArmCacheEnabled)).Returns("1");
+
+            using (var env = new TestScopedEnvironmentVariable(_vars))
+            {
+                var syncResult = await _functionsSyncManager.TrySyncTriggersAsync();
+                Assert.True(syncResult.Success, "SyncTriggers should return success true");
+
+                var logs = _loggerProvider.GetAllLogMessages();
+                var log = logs.First();
+                int startIdx = log.FormattedMessage.IndexOf("Content=") + 8;
+                int endIdx = log.FormattedMessage.LastIndexOf(')');
+                var triggersLog = log.FormattedMessage.Substring(startIdx, endIdx - startIdx).Trim();
+                var logObject = JObject.Parse(triggersLog);
+
+                // just verifying here that we're logging to both loggers, to ensure
+                // we're writing system logs
+                var trace = _traceWriter.GetTraces().First();
```

Comment:
I'm just going to verify this one, to make sure the logger is set up properly. I don't want to duplicate the rest of the tests in this file.